app/uk/gov/hmrc/soletraderidentificationfrontend/views/helpers/TitleHelper.scala

```diff
@@ -18,9 +18,17 @@ package uk.gov.hmrc.soletraderidentificationfrontend.views.helpers
 
 import play.api.data.Form
 import play.api.i18n.Messages
+import uk.gov.hmrc.soletraderidentificationfrontend.config.AppConfig
+import uk.gov.hmrc.soletraderidentificationfrontend.models.PageConfig
 
 object TitleHelper {
-  def title(titleMessage: String, form: Form[_])(implicit messages: Messages): String =
-    if(form.hasErrors) messages("error.title-prefix") + titleMessage
+  def title(titleKey: String, pageConfig: PageConfig, form: Form[_])(implicit messages: Messages, appConfig: AppConfig): String =
+    title(titleKey, pageConfig, form.hasErrors)
+
+  def title(titleKey: String, pageConfig: PageConfig, isAnErrorPage: Boolean = false)(implicit messages: Messages, appConfig: AppConfig): String = {
+    val titleMessage: String = s"${messages(titleKey)} - ${pageConfig.optServiceName.getOrElse(appConfig.defaultServiceName)} - ${messages("service.govuk")}"
```

Comment:
Hey Refaque .... I changed your implementation with ..... and the build is still green.... it means we are not testing the scenario in case we read the service name from the pageConfig. 
<img width="1278" alt="Screenshot 2022-04-22 at 16 26 01" src="https://user-images.githubusercontent.com/4958044/164745898-7f27326b-af36-4cf4-97d0-392f81c5e9a6.png">